src/main/webapp/WEB-INF/js/cartProductsList.js

```diff
@@ -0,0 +1,43 @@
+$(document).ready(() => {
+    var productsList = JSON.parse(localStorage.getItem("productsList"));
+    for (let cartProduct of productsList) {
+        let tableRow = $("<tr id ='" + cartProduct.productId + "'>" +
+            "<td>" + cartProduct.productName + "</td>" +
+            "<td>" + cartProduct.price + "</td>" +
+            "</tr>")
+            .data("productId", cartProduct.productId);
```

Comment:
Currently we get ids of products from productsList in the localStorage, so we don't need to keep it as it nor add it in `$.data()`.